### ISSUE TITLE
save a message for confirm box?

### ISSUE THREAD

**Issue by KeithMeurer** [jneilliii/OctoPrint-MQTTPublish#3]
How hard would it be to add a place to put a message for the confirm popup instead of the default? I would like to have it say "Are you sure you want to turn off the printer?" instead of "Continue sending message off to topic octomsv/cmd/pwr?"

**Comment by jneilliii** [comment]
It wouldn't be too difficult. Just curious, what type of devices are you using with what firmware? I have other tasmota plugins that do exactly what you're requesting, both mqtt and standard web variants, but your topic looks different to the tasmota default commands.

**Comment by KeithMeurer** [comment]
I'm actually pushing it to OpenHAB running on a Raspberry Pi 3. That's why I had flexibility in my topic names. The root name is octomsv so that I can have an additional octoprusa topic in the near future. ;-)

Thanks for this, I would love to contribute but the plugin framework looks like it would take me more cycles to digest than I have available. I'll take a look at your commits to see if it would help me contribute in the future.  Thanks!

**Comment by jneilliii** [comment]
My commits are typically a bunch of debug testing, so they may be hard to follow. For this change it's just adding an additional field to the topic array in default settings and incorporating that into the settings template. I can have this done in a couple of days.

**Comment by KeithMeurer** [comment]
Cool, no rush!

**Comment by KeithMeurer** [comment]
BTW, thanks for the Tasmota tip. I got a 4 pack in today, they are nice.

**Comment by jneilliii** [comment]
Yeah.  I did the same recently and the newer versions require you to solder in a header and flash over usb serial.  The OTA flashing is not possible in the newer released devices because sonoff hardened against that process for some reason.

**Comment by KeithMeurer** [comment]
I didn't realize you could flash the original OTA before. I went straight to the header.  Good stuff though, much cheaper than a zwave for a simple lamp switch.
